assets/src/scss/layout/navbar/_navigation-mobile-menu.scss

```diff
@@ -0,0 +1,226 @@
+$menu-footer-height: var(--navigation-mobile-footer--height, 108px);
+
+#navigation-mobile-menu {
+  position: fixed;
+  display: flex;
+  flex-direction: column;
+  justify-content: flex-start;
+  top: 0;
+  left: 0;
+  inset-inline-start: -100%;
+  padding: 18px $sp-3;
+  z-index: 99999;
+  overflow: hidden;
+  background: var(--top-navigation--background, $dark-blue);
+  height: 100%;
+
+  &.open {
+    inset-inline-start: 0;
+    transition: inset-inline-start 0.2s ease-out;
+
+    @supports not (inset-inline-start: 0) {
+      transform: translateX(0);
+      transition: transform 0.2s ease-out;
+
+      html[dir="rtl"] & {
+        transform: translateX(0);
+      }
+    }
+
+    ~ .top-navigation-overlay {
+      display: block;
+
+      @include large-and-up {
+        display: none;
+      }
+    }
+  }
+
+  ul {
+    padding: 0;
+    list-style: none;
+  }
+
+  .admin-bar & {
+    @include medium-and-less {
+      top: 32px;
+      height: calc(100% - 32px);
+    }
+
+    @include small-and-less {
+      top: 46px;
+      height: calc(100% - 46px);
+    }
+  }
+
+  @include small-and-less {
+    width: 100vw;
+    inset-inline-start: -100vw;
+
+    @supports not (inset-inline-start: -100vw) {
+      transform: translateX(-100vw);
+      transition: transform 0.2s ease-in;
+
+      html[dir="rtl"] & {
+        transform: translateX(100vw);
+      }
+    }
+  }
+
+  @include small-to-large {
+    width: 400px;
+    inset-inline-start: -400px;
+    transition: inset-inline-start 0.2s ease-in;
+
+    @supports not (inset-inline-start: -400px) {
+      transform: translateX(-400px);
+      transition: transform 0.2s ease-in;
+
+      html[dir="rtl"] & {
+        transform: translateX(400px);
+      }
+    }
+  }
+
+  @include large-and-up {
+    display: none;
+  }
+}
+
+.navigation-mobile-menu-header {
+  display: flex;
+  flex-direction: column;
+  flex-shrink: 0;
+  border-bottom: var(--top-navigation--separation, 1px solid transparentize($white, 0.5));
+  padding-bottom: $sp-2;
+
+  .site-logo {
+    display: block;
+    width: fit-content;
+    margin-inline-start: 0;
+    margin-bottom: $sp-1x;
+    line-height: 1;
+
+    img {
+      height: 23px;
+    }
+  }
+
+  .nav-languages {
+    width: 100%;
+    margin-top: $sp-1x;
+
+    ul {
+      margin-inline: 0;
+      padding: 0;
+      margin-bottom: 0;
+    }
+  }
+}
+
+.nav-wrapper {
+  overflow-y: scroll;
+
+  .nav-items {
+    width: 100%;
+    padding: 0;
+  }
+
+  @include scrollbar-layout;
+}
+
+.navigation-mobile-menu-footer {
+  display: flex;
+  justify-self: flex-end;
+  flex-shrink: 0;
+  margin-top: auto;
+  width: 100%;
+  padding: $sp-3 $sp-3 0;
+  border-top: var(--top-navigation--separation, 1px solid transparentize($white, 0.5));
+
+  .btn-donate {
+    width: 100%;
+  }
+}
+
+
+// All styles related to the menu item
+.navigation-mobile-menu-item {
+  color: inherit;
+  font-size: $font-base-size;
+  font-weight: 700;
+  width: 100%;
+
+  a.nav-link {
```

Comment:
There's no green underline on hover, is that expected? The active item does have it though